Orange/widgets/visualize/owmosaic.py

```diff
@@ -294,7 +294,7 @@ def initCombos(self, data):
         self.attr3Combo.addItem("(None)")
         self.attr4Combo.addItem("(None)")
 
-        for attr in data.domain:
+        for attr in chain(data.domain.attributes, data.domain.metas):
```

Comment:
Wasn't the pull request supposed to just add meta variables?
Why have we switched from `data.domain` to `data.domain.attributes` which does not have the class variables?
Probably not intended... @kernc @janezd ?